### PR TITLE
Fix the create_json_schema issue

### PR DESCRIPTION
**Pull request by mkanoor**
The swagger yml had the datatype marked as string instead of object

### CODE REVIEW

**Comment by mkanoor** [comment]
@Fryguy 
Please review

File: .gitignore [hunk 1]
```diff
@@ -5,7 +5,6 @@
 *.rbc
 /.config
 /coverage/
-/Gemfile.lock
```
Comment (Fryguy) [inline .gitignore]: Please don't change the gitignore

File: client-meta.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+	"moduleName": "TopologicalInventoryApiClient",
+	"gemName": "topological_inventory-api-client",
+	"gemHomepage" : "https://github.com/ManageIQ/topological_inventory-api-client-ruby",
+	"gemAuthor" : "manageiq_authors",
```
Comment (Fryguy) [inline client-meta.json]: Prefer "ManageIQ Authors"

File: spec/api/default_api_spec.rb [hunk 3]
```diff
@@ -153,6 +153,20 @@
...
+  # List Tags for ContainerGroup
+  # Returns an array of Tag objects
+  # @param id ID of the resource
+  # @param [Hash] opts the optional parameters
+  # @option opts [Integer] :limit The numbers of items to return per page.
+  # @option opts [Integer] :offset The number of items to skip before starting to collect the result set.
+  # @return [TagsCollection]
+  describe 'list_container_group_tags test' do
+    it 'should work' do
+      # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
```
Comment (Fryguy) [inline spec/api/default_api_spec.rb:166]: My new favorite spec 😂

**Review by Fryguy — COMMENTED** [review]
Minor stuff

**Comment by mkanoor** [comment]
@Fryguy Please review changes made

**Review by carbonin — APPROVED** [review]

**Pull request merged**